alacritty/src/display/mod.rs

```diff
@@ -637,6 +634,14 @@ impl Display {
             config.window.dynamic_padding,
         );
 
+        // Queue renderer update if terminal dimensions/padding changed.
+        if new_size != self.size_info {
+            let renderer_update = self.pending_renderer_update.get_or_insert(Default::default());
+            renderer_update.resize = true;
```

Comment:
It's not a resize anymore, since size could be the same, so something to `size_info` should be used.

Reply:
What do you mean?

Reply:
I mean that `render_update.resize` should be named `render_update.size_info_changed`, since resize is about dimensions but we could have a change without dimensions change.

Reply:
I think it's fine since it determines if a resize should be done, not that something was resized.